bundles/persistence/org.openhab.persistence.mapdb/src/main/java/org/openhab/persistence/mapdb/internal/MapDBItem.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.persistence.mapdb.internal;
+
+import java.io.Serializable;
+import java.text.DateFormat;
+import java.util.Date;
+
+import org.openhab.core.persistence.HistoricItem;
+import org.openhab.core.types.State;
+
+/**
+ * This is a Java bean used to persist item states with timestamps in the database.
+ * 
+ * @author Jens Viebig
+ * @since 1.0.0
+ *
+ */
+public class MapDBItem implements HistoricItem, Serializable {
+
+	/**
+	 * 
+	 */
```

Comment:
remove this empty comment of (even better) briefly explain the purpose of this member